pkg/proc/proc_test.go

```diff
@@ -2934,3 +2934,21 @@ func TestRecursiveNext(t *testing.T) {
 		}
 	})
 }
+
+func TestIssue877(t *testing.T) {
+	if runtime.GOOS != "darwin" && testBackend == "lldb" {
```

Comment:
Can you add a comment describing what the test does "Tests that environment variables starting with DYLD_" are passed to the target on macOS". Thank you.